packages/legend-storage/src/SDLC.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ProjectGAVCoordinates } from './DependencyUtils.js';
+
+export class LegendSDLC implements ProjectGAVCoordinates {
```

Comment:
this file being here looks wrong to me, perhaps let's have an abstract file called `GraphDataOrigin` and make `LegendSDLC extends` that, we don't even need to say `implements ProjectGAVCoordinates` doesn't buy us any good